ranger-plugin/scripts/enable-ranger-plugin.sh

```diff
@@ -104,8 +157,18 @@ function get_hawq_password() {
   done
 }
 
+function sync_rps_configuration() {
+  if [[ -d $GPHOME/ranger/etc/ && ! -z $HAWQ_STANDBY_HOST ]]; then
+    cmd="scp $GPHOME/ranger/etc/* $HAWQ_STANDBY_HOST:$GPHOME/ranger/etc/"
```

Comment:
are we sure we want to copy the files here as part of this script ? I'd say it would be better to have it working on a single node only and do any cluster-level operations in "hawq" management scripts. Also, in the future config files will be managed by Ambari. I'd say it even could be ok to ask users to copy the files to standby manually, especially since when failover occurs, they will need to provision a new standby and will have to copy config files there manually anyways.